### DISCUSSION TITLE
Navigation select waypoint to start rerouting

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#178] [Q&A] [answered]
When we start navigation away from the route, until now we could select to reroute to: Start, Nearest waypoint, End.
We can make this function better and more user friendly. :slightly_smiling_face: 

When we start navigation away from the route, we can **select the waypoint** to enter the route and the first rerouting:
What do you think?

_Remember that the rerouting in Cruiser does **NOT** change the route!_

<img src="https://github.com/devemux86/cruiser/assets/3484020/13ce6be7-c710-49ca-ae35-f3fc3d9913cd" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/b57cdeb1-45f4-4463-80c9-f1cf463500fa" width="50%"/>

**Comment by MotoUKRider** [comment]
*Small minor teeny tiny little suggestion*: Make Start+End icon colours match the map+Waypoints list?!
(or would that impact light/dark theme later on?)

**Reply by devemux86** [reply]
Indeed we could use the real (numbered) icons of the waypoints?

They work in both light + dark themes.
Like the waypoints list via the menu "Routing | Waypoints".

**Reply by devemux86** [reply]
> Make Start+End icon colours match the map+Waypoints list?!

Done! Please see the updated images above: https://github.com/devemux86/cruiser/discussions/178#discussion-5225585

There will be a similar color appearance on the other screens showing waypoint selection:
- Action "Destination" (tap top-left navigation panel)
- Next waypoint panel (long press top-right distance panel or top waypoint panel)

**Comment by devemux86** [comment, marked as answer]
When we start navigation away from the route, now we can select the waypoint for the first reroute.
Available in [Cruiser 3.0.21](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

_Remember that the rerouting in Cruiser does **NOT** change the route!_

Also all waypoint screens have been redesigned displaying colored waypoint types and numbers.
See the images above: https://github.com/devemux86/cruiser/discussions/178#discussion-5225585
